### PR TITLE
add support for min and max items - fix sprintf error

### PR DESCRIPTION
**Pull request by tjungblu**
Hey there,

I added support for min and max items in an array (also nested). I also found a typo in the use of Sprintf (it was sprinf).
Let me know if there's anything to change, happy to update.

Cheers,
Thomas

### CODE REVIEW

File: pkg/codegen/model.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package codegen
 
 import (
+	"github.com/atombender/go-jsonschema/pkg/schemas"
```
Comment (arjunrn) [inline pkg/codegen/model.go:4]: Don't know about the formatting requirements here but this should probably go after the standard library imports.

Comment (tjungblu) [reply pkg/codegen/model.go:4]: is there a way to tell Goland about this? govet also doesn't pick that up somehow

Comment (arjunrn) [reply pkg/codegen/model.go:4]: Settings->Editor->Code Style->Go and there you can specify import order and switch over from `gofmt` to `goimports`.

**Comment by atombender** [comment]
Awesome, thanks!

**Pull request merged**